effectsize_supplement.Rmd

```diff
@@ -0,0 +1,281 @@
+
+The [simple effect size exemplar](effectsize_example.html#exemplar_simple_effect_size) demonstrates one common technique for esitmating mean differences in resposne time and the uncertainty around them (Student's t confidence intervals). This supplement demonstrates several possible approaches one might take to calculate differences in response time, and compares them. It is not intended to be exhaustive.
+
+
+## Libraries needed for this analysis
+
+```{r setup, warning = FALSE, message = FALSE}
+library(tidyverse)
+library(forcats)    # for fct_...()
+library(broom)      # for tidy()
+library(ggstance)   # for geom_pointrangeh(), stat_summaryh()
+
+# for brm(). Requires rstan; see here for installation instructions: 
+# https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started
+library(brms)       
+
+# for mean_qi(). Use devtools::install_github("mjskay/tidybayes") to install.
+library(tidybayes)
+
+# requires `import` package to be installed: use install.packages("import")
+import::from(MASS, mvrnorm)
+```
+
+```{r boilerplate, include = FALSE}
+format_num <- function(nums, sigdigits = 3) gsub("\\.$", "", formatC(nums, sigdigits, format = "fg", flag="#"))
+```
+
+
+## Data
+
+We will use the same data as the simple effect size exemplar: 
+
+```{r data_generation}
+set.seed(12)
+n <- 20
+data <- tibble(
+  group = rep(c("A", "B"), each = n),
+  completion_time_ms = c(
+    rlnorm(n, meanlog = log(170), sdlog = 0.3),
+    rlnorm(n, meanlog = log(50), sdlog = 0.4)
+  )
+)
+```
+
+See that exemplar for more information.
+
+
+## Calculating simple effect size
+
+### Approach 1: Difference in means with Student's t confidence interval
+
+This is the approach used in the exemplar. While the response distributions are non-normal, the sampling distribution of the difference in means will still be defined on $(-\infty, +\infty)$ and approximately symmetrical [citation needed / double check this], so we can compute a *Student's t distribution confidence interval* for the difference in means.
+
+```{r t_test}
+t_result <- 
+  t.test(completion_time_ms ~ group, data = data) %>%
+  tidy()    # put result in tidy tabular format
+t_result
+```
+
+The `tidy()`ed output of the `t.test()` function includes an estimate of the mean difference in milliseconds (`estimate`) as well as the lower (`conf.low`) and upper (`conf.high`) bounds of the 95% confidence interval. 
+
+### Approach 2a: Ratio of medians with Student's t confidence interval on log-scale
+
+For responses that are assumed to be log-normal, one alternative is to calculate the mean difference on the log scale. Because the mean on the log scale of a log-normal distribution is equal to the median of the distribution of the untransformed responses, this is equivalent to calculating a ratio of medians on the untransformed scale (in this case, a ratio of median response times). See the [data transformation](data_transformation) guideline for more information.
```

Comment:
Fair, this does rely on the assumption that the distribution on the log scale is symmetrical. I had the same argument with Jeff Heer when we wrote the Beyond Weber's Law paper. I'll switch it to geometric mean. :)